android/src/main/java/com/adyenreactnativesdk/component/BaseModule.kt

```diff
@@ -48,6 +48,17 @@ abstract class BaseModule(context: ReactApplicationContext?) : ReactContextBaseJ
         }
     }
 
+    protected fun getPaymentMethod(
+        paymentMethodsResponse: PaymentMethodsApiResponse,
+        paymentMethodNames: Set<String>
+    ): PaymentMethod? {
+        for (currentPaymentMethod in paymentMethodsResponse.paymentMethods ?: emptyList())
```

Comment:
This can be simplified by 
```
return paymentMethodsResponse.paymentMethods?.firstOrNull { paymentMethodNames.contains(it.type) }
```

Reply:
fixed [6788eb9](https://github.com/Adyen/adyen-react-native/pull/105/commits/6788eb96b96f27e0ddb86d2328a52f4d95d5800d)